src/plugins/discover/public/application/main/components/top_nav/get_top_nav_links.tsx

```diff
@@ -60,6 +65,55 @@ export const getTopNavLinks = ({
     testId: 'discoverAlertsButton',
   };
 
+  /**
+   * Switches from ES|QL to classic mode and vice versa
+   */
+  const esqLDataViewTransitionToggle = {
+    id: 'esql',
+    label: isEsqlMode
+      ? i18n.translate('discover.localMenu.localMenu.switchToClassicTitle', {
+          defaultMessage: 'Switch to classic',
+        })
+      : i18n.translate('discover.localMenu.localMenu.tryESQLTitle', {
+          defaultMessage: 'Try ES|QL',
+        }),
+    emphasize: true,
+    iconType: 'editorRedo',
+    description: isEsqlMode
+      ? i18n.translate('discover.localMenu.localMenu.switchToClassicTitle', {
+          defaultMessage: 'Switch to classic',
+        })
+      : i18n.translate('discover.localMenu.tryESQLDescription', {
+          defaultMessage: 'Try ES|QL',
+        }),
```

Comment:
Does it need the same description as the title?

Reply:
Removed here https://github.com/elastic/kibana/pull/188898/commits/fcea60218ca2cb9ed2ba997763583d60e3a72729